sdk/storage/azure-storage-blob-changefeed/azure/storage/blob/changefeed/_models.py

```diff
@@ -84,9 +96,21 @@ def __init__(self, client, page_size, start_time=None, end_time=None, cf_cursor=
         self._segment_paths_generator = None
         self.current_segment = None
         self.start_time = start_time
-        self.end_time = end_time
-        self._initialize(cf_cursor=cf_cursor)
-        self.cursor = None
+
+        # the end time is in str format
+        end_time_in_cursor = cf_cursor['EndTime'] if cf_cursor else None
+        # convert the end time in str format to a datetime object
+        end_time_in_cursor_obj = \
+            datetime.strptime(end_time_in_cursor, '%Y-%m-%dT%H:%M:%S+00:00') if end_time_in_cursor else None
+        # self.end_time is in datetime format
+        self.end_time = end_time or end_time_in_cursor_obj
```

Comment:
In C# and JAVA we either accept token OR start and end time. Should we make these mutually exclusive here as well ?

Reply:
I don’t think we want to make it mutual exclusive.
It’s possible that the user doesn’t want to specify an end time when listing events at first, then the user started using the continuation token that doesn’t have an end time, while he wants to add an end time. It’s also possible he wants to overwrite the end time.

Reply:
I think that this would only complicate usage of this feature and number of corner cases grows if we allow that.
I think user should:
 - use start_time and end_time (optional) to define boundaries of iteration
 - use continuation_token to progress/resume already defined listing of events in frozen context.

Reply:
ahhh while I think it's just to give customers more choice. We never stop adding features because that will cause more random cases to look out right?

Reply:
I think adding a feature when its asked for is a little different than this case. I think to start off, we need to make using CF as simple as possible. Should a customer request this ability, we can always have a discussion about it then.

Reply:
Here is the python changefeed interface, `list_changes(self, **kwargs):`
end_time is in **kwargs, it will not add any complexity. user need to specify end_time=***** then it will be applied.